src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-bucket-encryption.ts

```diff
@@ -12,26 +13,19 @@ enum SecretEngines {
   Transit = 'transit'
 }
 
-enum sseS3 {
-  SSE_S3 = 'AES256'
-}
-
-enum sseKms {
-  SSE_KMS = 'aws:kms'
+export enum ENCRYPTION_TYPE {
+  SSE_S3 = 's3',
+  SSE_KMS = 'kms'
```

Comment:
```suggestion
  S3 = 's3',
  KMS = 'kms'
```
encryption type is enough to depict

Reply:
I think its fine. We are using the name as per the category 
![image](https://github.com/user-attachments/assets/fffd749a-e2c1-4d85-9ea1-fdeca27aa20c)

Reply:
We can give its S3 and KMS in UI as well. We dont need to call "SSE"

Reply:
Also we need one line helper text for each. please connect with doc team. Thanks

Reply:
@bhkaur25 ^^

@afreen23 let it be SSE-S3 for now, it needs discussion with @likid0

Reply:
Ack @cloudbehl 
(Non-blocking comment:)
Its ts not good to have abbrev embedded in UI.
Thinking more , the form modal title should be "Configure RGW Server Side Encryption" and then options as "S3" and  "KMS". `SSE_S3 an SSE_KMS` does not explain well here.
Even if we dont change title we may add more help texts here too, thanks
![image](https://github.com/user-attachments/assets/b0e401bd-575a-4e4a-840c-e7a4aeb71836)

Reply:
thanks @likid0